homeworks/nemethmate/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,22 @@
+# Adatok bekérése
+city = input("Kérlek add meg a várost: ")
+rent = float(input("Kérlek add meg az albérlet árát (USD): "))
+
+# Feltételek
+if city.lower() == "new york" or city.lower() == "san francisco":
```

Comment:
a lower()-t meghívhatnád az első soron is, és így itt nem lenne szükség arra hogy kétszer külön is meghívd.
valamint használhatnál membership operator-okat hogy egyszerűbb legyen:

`if city in ["new york", "san francisco"]:`